ambry-messageformat/src/main/java/com/github/ambry/messageformat/MessageFormatRecord.java

```diff
@@ -1817,7 +1816,20 @@ public static BlobData deserializeBlobRecord(CrcInputStream crcStream) throws IO
       long crc = crcStream.getValue();
       long streamCrc = dataStream.readLong();
       if (crc != streamCrc) {
-        logger.error("corrupt data while parsing blob content expectedcrc {} actualcrc {}", crc, streamCrc);
+        StringBuilder sb = new StringBuilder();
+        sb.append("ExpectedCrc: ")
```

Comment:
Can you change ExpectedCrc to ComputedCrc
and ActualCrc to ReceivedCrc ?
The terms expected and actual are super confusing